.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "semi": true,
+  "tabWidth": 4,
+  "useTabs": false,
+  "printWidth": 100,
+  "singleQuote": true,
```

Comment:
Can we use single and double quotes?

Reply:
No, this rule indicates that single quotes will be used instead of double quotes.